xml/nvset-4.0.rng

```diff
@@ -44,12 +44,7 @@
       <zeroOrMore>
         <element name="nvpair">
           <choice>
-            <group>
-              <attribute name="id-ref"><data type="IDREF"/></attribute>
-              <optional>
-                <attribute name="name"><text/></attribute>
-              </optional>
-            </group>
+            <attribute name="id-ref"><data type="IDREF"/></attribute>
```

Comment:
This commit broke the `nvpair-id-ref` scheduler regression test output. It's ignored/broken in a lot of places that assume `name` occurs if and only if `id` occurs. But `populate_hash()` considers it.

I'd still prefer to drop this feature -- partly because it's broken in places, and partly because it's rarely used, it's undocumented, and it complicates the transformation. Although, since `populate_hash()` does something with it, we may still want a transformation to convert these to nvpairs with a different ID based on the `name` attribute.

If we must keep this feature, we can try to work around it in the transformation pipeline. But a workaround would be error-prone: it's difficult to be sure we're covering and testing all possible cases. If we must, we can try.